src/askContainer/treeViews/helpView.ts

```diff
@@ -1,23 +1,17 @@
-import * as vscode from 'vscode';
-import { PluginTreeItem, Resource } from '../../runtime';
+import * as vscode from "vscode";
```

Comment:
nit: Is this a style preference coming from your tslint or eslint file? I don't see the compiler or vscode complaining about this in my local source code.

Reply:
Yeah, I used the prettier to fix some lint issues and it automatically change the single quote to double quote. Right now we don't have specific rule for this quote style, please ignore those changes.

Reply:
Either add in your prettier config in the source code or better to reset those changes so that the diff is cleaner. Your call.

Reply:
Got it, update the prettier config to always use single quote. But it will change existing double quotes to single quote as well.